content/developers/dogstatsd/data_types.md

```diff
@@ -0,0 +1,309 @@
+- `mywebsite.page_render.time.95percentile` - the 95th percentile render time
+
+Under the hood, DogStatsD treats timers as histograms. Whether you timers or histograms, you'll be sending the same data to Datadog.
+
+### Sets
+
+Sets are used to count the number of unique elements in a group, for example, the number of unique visitors to your site:
+
+For Python:
+```python
+
+def login(self, user_id):
+    # Log the user in ...
+    statsd.set('users.uniques', user_id)
+```
+
+For Ruby:
+```ruby
+def login(self, user_id)
+    # Log the user in ...
+    statsd.set('users.uniques', user_id)
+end
+```
+
+Learn more about the [Histogram type in the Metrics documentation][6].
+
+## Metric option: sample rates
+
+Since the overhead of sending UDP packets can be too great for some performance
+intensive code paths, DogStatsD clients support sampling,
+i.e. only sending metrics a percentage of the time. The following code sends
+a histogram metric only about half of the time:
+
+```python
+
+dog.histogram('my.histogram', 1, sample_rate=0.5)
+```
+
+Before sending the metric to Datadog, DogStatsD uses the `sample_rate` to
+correct the metric value, i.e. to estimate what it would have been without sampling.
+
+**Sample rates only work with counter, histogram, and timer metrics.**
+
+Learn more about the [Rates in the Metrics documentation][7].
+
+## Events
+
+DogStatsD can emit events to your Datadog event stream. For example, you may want to see errors and exceptions in Datadog:
+
+```python
+
+from datadog import statsd
+
+def render_page():
+  try:
+    # Render the page...
+    # ..
+  except RenderError as err:
+    statsd.event('Page render error!', err.message, alert_type='error')
+```
+
+## Service Checks
+
+DogStatsD can send service checks to Datadog. Use checks to track the status of services your application depends on:
+
+For Python:
+```python
+
+from datadog.api.constants import CheckStatus
+
+# Report the status of an app.
+name = 'web.app1'
+status = CheckStatus.OK
+message = 'Response: 200 OK'
+
+statsd.service_check(check_name=name, status=status, message=message)
+```
+
+For Ruby:
+```ruby
+# Report the status of an app.
+name = 'web.app1'
+status = Datadog::Statsd::OK
+opts = {
+  'message' => 'Response: 200 OK'
+}
+
+statsd.service_check(name, status, opts)
+```
+
+After a service check has been reported, you can use it to trigger a [custom check monitor][3].
+
+## Tagging
+
+You can add tags to any metric, event, or service check you send to DogStatsD. For example, you could compare the performance of two algorithms by tagging a timer metric with the algorithm version:
+
+```python
+
+@statsd.timed('algorithm.run_time', tags=['algorithm:one'])
+def algorithm_one():
+    # Do fancy things here ...
+
+@statsd.timed('algorithm.run_time', tags=['algorithm:two'])
+def algorithm_two():
+    # Do fancy things (maybe faster?) here ...
+```
+
+Since tagging is an extension to StatsD, use a [DogStatsD client library][1].
```

Comment:
`(...) use a [DogStatsD client library][1] depending of your application language` ?